bundles/org.openhab.binding.lcn/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -277,6 +277,14 @@
 		<item-type>Contact</item-type>
 		<label>Binary Sensor</label>
 		<autoUpdatePolicy>veto</autoUpdatePolicy>
+		<config-description>
+			<parameter name="invertState" type="boolean">
+				<label>Invert state</label>
```

Comment:
Words in labels should be capitalized. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions